python/tvm/relay/op/tensor.py

```diff
@@ -244,3 +244,35 @@ def concat(*args):
     """
     tup = Tuple(list(args))
     return _make.concat(tup)
+
+
+def clip(a, a_min, a_max):
+    """Clip the elements in `a` between `min` and `max`.
+
+    Parameters
+    ----------
+    a : relay.Expr
+        The input tensor.
+    a_min : relay.Constant
```

Comment:
Although relay.Constant is more general in this case, let us just use float and make a_min and a_max attributes for now, this is the most common usecase of clip

Reply:
Do you mean that `a_min` and `a_max` should be python attributes? That deviates from numpy's function signature and also makes it impossible to enforce that `a_min` and `a_max` must be provided.

Reply:
I mean make a_min and a_max as part of Attrs, see e.g. https://github.com/dmlc/tvm/blob/master/include/tvm/relay/attrs/transform.h#L17